contact.html

```diff
@@ -4,8 +4,37 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Contact</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web&display=swap" rel="stylesheet">
 </head>
 <body>
+    <address>
+        <h1>Say hello!</h1>
+        <img src="https://media.giphy.com/media/3oz8xTAJIQD6JWfTUc/giphy.gif" alt="wave">
+        <p>By Phone: 972-123-4567</p>
+        <form action="mailto:jrios1991@gmail.com" method="POST">
+            <p>Or by Email</p>
+            <label for="fname">First Name
```

Comment:
The 'for' attribute is intended to reference the element to which the label belongs.  In this case, there is no element with the 'id' or 'name' "fname".